runtime/tywrap_bridge_core.py

```
@@ -0,0 +1,865 @@
+        'instances': len(instances),
+    }
+
+
+def dispatch_request(
+    msg,
+    instances,
+    *,
+    bridge,
+    pid,
+    force_json_markers,
+    allow_nan=False,
```

Ruff (0.15.14)

[warning] Unused function argument: `allow_nan`

(ARG001)